generate_samples.py

```diff
@@ -278,17 +287,60 @@ def generate_samples_onnx(
             else:
                 wav_path = output_dir / f"{sample_idx}.wav"
 
-            wav_file: wave.Wave_write = wave.open(str(wav_path), "wb")
-            voice.synthesize_wav(
-                next(texts),
-                wav_file=wav_file,
-                syn_config=SynthesisConfig(
+            text_input = next(texts)
+
+            if phoneme_input:
+                # For ONNX models with phoneme input, build phoneme IDs manually
+                phonemes = [p for p in list(text_input)]
```

Comment:
The text needs to be normalized first: `unicodedata.normalize("NFD", text_input)`